lib/recommendations/getRecommendationsForUser.ts

```diff
@@ -1,14 +1,42 @@
 import { doc, getDoc } from "firebase/firestore";
 
+import fetchServer from "@/lib/fetch/fetchServer";
 import { profilesCol } from "@/lib/firestore";
 
 import getRecommendationsBySeed from "./getRecommendationsBySeed";
 
 const DEFAULT_ARTIST = "4NHQUGzhtTLFvgF5SZesLK";
 const DEFAULT_TRACK = "0c6xIDDpzE81m2q797ordA";
 
-const getRecommendationsForUser = async (userId: string, limit?: number) => {
-  const profile = await getDoc(doc(profilesCol, userId));
+const getProfile = async (userId: string) => getDoc(doc(profilesCol, userId));
+
+const getRecommendedArtistData = async (id: string, limit?: number) => {
+  const data = await fetchServer<SpotifyApi.ArtistsRelatedArtistsResponse>(
+    `https://api.spotify.com/v1/artists/${id}/related-artists`
+  );
+  return data.artists.slice(0, limit);
+};
+
+export const getRecommendedArtists = async (userId: string, limit?: number) => {
```

Comment:
move this to another file (you can also probably merge it with the `getRecommendedArtistData` function I don't think they need to be separate)